src/member/password/hooks.test.ts

```diff
@@ -0,0 +1,52 @@
+import { StatusCodes } from 'http-status-codes';
+import nock from 'nock';
+import { afterEach, describe, expect, it } from 'vitest';
+
+import { UNAUTHORIZED_RESPONSE } from '../../../test/constants.js';
+import { mockHook, setUpTest } from '../../../test/utils.js';
+import { memberKeys } from '../../keys.js';
+import { buildGetPasswordStatusRoute } from './routes.js';
+
+const { hooks, wrapper, queryClient } = setUpTest();
+describe('Member Hooks', () => {
+  afterEach(() => {
+    queryClient.clear();
+    nock.cleanAll();
+  });
+
+  describe('usePasswordStatus', () => {
+    const route = `/${buildGetPasswordStatusRoute()}`;
+    const hook = () => hooks.usePasswordStatus();
```

Comment:
Do we need to wrap the hook ?
```suggestion
    const hook = hooks.usePasswordStatus;
```